.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Just a reminder that text in these brackets is not visible in the rendered output
+You also do *not* need to use this template. Its just a suggestion :)
+-->
+
+
+
+## Changes
+<!-- Please describe what this Pull request adds or changes. Possibly with  images -->
+
+## Todo before merging
+<!-- Add custom todos if deemed necessary to give others an overview on how far this PR is progressing -->
+
+<!-- Did you add UI text? 
+No? Just check or remove the box
+Yes? Make sure you remove hardcoded strings with translations -->
```

Comment:
replace

Can we link to the contribution guidelines here, I think there is something related to translation strings in there, or not?